src/pages/landing/landing-page.ts

```diff
@@ -2,86 +2,89 @@ import { LitElement, css, html } from "lit";
+import { WAKE_WORDS, ICON_CHEVRON_SLOTTED, PAGE_STYLES } from "../../const";
 
 @customElement("landing-page")
 export class LandingPage extends LitElement {
   @property() public selectWakeWord!: (wakeWord: string) => void;
 
   render() {
     return html`
-      <card-layout header="Wake Word Collective">
+      <card-layout>
         <div class="card-content">
           <p>
-            With
-            <a href="https://www.home-assistant.io" target="_blank"
-              >Home Assistant</a
-            >
-            we set the goal to let anyone control their home in their own
-            language with a privacy focused voice assistant. We have developed
-            multiple local technologies to make this possible. One of them is a
-            novel wake word engine,
-            <a href="https://github.com/kahrendt/microWakeWord" target="_blank"
-              >microWakeWord</a
-            >, that is open source and runs on cheap hardware.
-          </p>
-          <p>
-            It is important that voice assistants only listen when a user says
-            the wake word, but this is not an easy task! To increase the
-            accuracy of microWakeWord, we need recordings from people from all
-            genders, ages, and accents from all over the world saying wake
-            words.
-          </p>
-          <p>
-            This website allows you to record yourself with your phone or
-            computer and contribute them to the collective. We're starting with
-            one wake word, and are working with the community to decide which to
-            add next.
+            Give us a minute of your time to improve our community-driven,
+            privacy-focused voice assistant, and we’ll break big tech’s
+            stranglehold on home voice control.
           </p>
+          <div class="note">
+            <p>
+              Specifically, we’re trying to improve our wake word engine, which
+              “wakes” the device to listen for more commands. Our open source
+              <a
+                href="https://github.com/kahrendt/microWakeWord"
+                target="_blank"
+                >microWakeWord</a
+              >
+              engine is an incredible, lightweight bit of code but requires
+              training with real voices to improve.
+            </p>
+            <p>
+              Everyone’s voice is unique, we need recordings of people of all
+              genders, ages, and accents from around the world saying this wake
+              word.
+            </p>
+          </div>
         </div>
-        <md-list>
-          ${Object.entries(WAKE_WORDS).map(
-            ([key, label]) => html`
-              <md-list-item
-                type="button"
-                @click=${() => {
-                  this.selectWakeWord(key);
-                }}
+        ${Object.entries(WAKE_WORDS).map(
+          ([key, label]) => html`
+            <div class="card-actions">
```

Comment:
Don't put card-actions inside the list. That element is not meant to be rendered multiple times